examples/netlink.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2018 Intel Corporation.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"flag"
+	"net"
+	"time"
+
+	"github.com/intel-go/nff-go/common"
+	"github.com/intel-go/nff-go/flow"
+	"github.com/intel-go/nff-go/packet"
+	"golang.org/x/sys/unix"
+
+	"github.com/vishvananda/netlink"
+)
+
+var lpm *packet.LPM
+
+// lpm struct binds ip with number and below
+// is a struct to get ip by binded value
+var portToIP []uint32
+
+// length of portToIp
+var lenP2IP uint32
+
+const (
```

Comment:
I think we should be consistent with the examples: make them external parameters with default values, as everywhere else.